src/pudl/metadata/fields.py

```diff
@@ -5925,7 +5925,7 @@
             "still charge from the grid)."
         ),
     },
-    "id_dc_coupled_tightly": {
+    "is_dc_coupled_tightly": {
```

Comment:
Given the other similar columns, this sure seemed like a typo, and it just happened to also be all null (apparently not related to the typo) so I updated the name. Still not sure why it's all null.